BE/src/main/java/com/example/be/business/dish/controller/DishController.java

```diff
@@ -24,23 +21,20 @@ public DishController(DishService dishService, CategoryRepository categoryReposi
         this.categoryRepository = categoryRepository;
     }
 
-    @GetMapping
-    public List<PlanningDataRequest> getPlanningData() {
-        return dishService.getPlanningData();
-    }
-
-//    @GetMapping
-//    public ApiResult<List<PlanningDataRequest>> getPlanningData() {
-//        return OK(dishService.getPlanningData());
-//    }
-
     @GetMapping("{id}")
     public ResponseEntity<DishDetailResponse> getDishDetail(@PathVariable("id") Long id) {
         return new ResponseEntity<>(new DishDetailResponse(dishService.getDishDetail(id)), HttpStatus.OK);
     }
 
-    @GetMapping("/test")
+    @GetMapping("/planning")
     public ResponseEntity<PlanningDataRequestByCategory> getDishesByCategory() {
         return new ResponseEntity<>(new PlanningDataRequestByCategory(dishService.getPlanningData()), HttpStatus.OK);
     }
+
+    @PostMapping("{id}/update")
+    public ResponseEntity<PlanningDataRequestByCategory> updateDish(@PathVariable("id") Long id, @RequestBody DishUpdateRequest dishUpdateRequest) {
+        dishService.updateDish(id, dishUpdateRequest.getPrice(), dishUpdateRequest.getBadge(), dishUpdateRequest.getDeliveryPriceOption(),
```

Comment:
저라면, `dishService` 에 `dishUpdateRequest` 를 집어넣고 `dishService` 에게 해당 DTO를 활용하여 어떻게 데이터를 가공하여 사용할 것인지에 대한 책임과 권한을 위임했을 것 같아요.
```suggestion
        dishService.updateDish(id, dishUpdateRequest);
```

Reply:
이번 프로젝트에서 service에 dto를 의도적으로 안 넣으려다 보니 지저분하게 된 것 같습니다 😭